README.md

```diff
@@ -49,6 +49,31 @@ Each benchmark will run until the target quality is reached and then stop, print
 
 Some these benchmarks are rather slow or take a long time to run on the reference hardware (i.e. 16 CPUs and one P100). We expect to see significant performance improvements with more hardware and optimized implementations. 
 
+
+# Running Instructions
```

Comment:
Can we make this a subsection of the "Runing benchmarks" and change the title to "Running with popper"